src/layouts/full_size_dual/default.scad

```diff
@@ -0,0 +1,27 @@
+include <../layout.scad>
+
+rt_arrow="\u2192";
+lt_arrow="\u2190";
+up_arrow="\u2191";
+dn_arrow="\u2193";
+menu="\u2261";
+enter=lt_arrow; //"\u2386" with supported Unicode Font
+backspace="BkSp"; //"\u232B" with supported Unicode Font
+
+// Keys are defined independently: [legend, horizontalSize, verticalSize, fontChg]
+full_size_dual = [[["Esc",1,1,-2], ["",-1,1,0], ["F1",1,1,-2], ["F2",1,1,-2], ["F3",1,1,-2], ["F4",1,1,-2], ["",-.5,1,0], ["F5",1,1,-2], ["F6",1,1,-2], ["F7",1,1,-2], ["F8",1,1,-2], ["",-.5,1,0], ["F9",1,1,-2], ["F10",1,1,-2], ["F11",1,1,-2], ["F12",1,1,-2],                         ["",-0.5,1,0], [["Print","Screen"],1,1,-3], [["Scroll", "Lock"],1,1,-3], [["Pause","Break"],1,1,-3]],
```

Comment:
New Methodology for defining layouts that allows for collocated definitions of the legend, horizontal/vertical size, and font size changes. This makes it so that there is less likely to have errors (I fat finger things a lot) with multiple tables.